python/paddle/fluid/tests/unittests/test_imperative_basic.py

```diff
@@ -640,6 +641,31 @@ def test_append_activation_in_dygraph3(self):
             res2 = fluid.layers.sigmoid(a)
             self.assertTrue(np.array_equal(res1.numpy(), res2.numpy()))
 
+    def test_append_activation_in_dygraph_use_mkldnn(self):
+        a_np = np.random.uniform(-2, 2, (10, 20, 30)).astype(np.float32)
+        helper = LayerHelper(
+            fluid.unique_name.generate("test"), act="relu", use_mkldnn=True)
+        func = helper.append_activation
+        with fluid.dygraph.guard():
+            a = fluid.dygraph.to_variable(a_np)
+            res1 = func(a)
+            res2 = fluid.layers.relu(a)
+            self.assertTrue(np.array_equal(res1.numpy(), res2.numpy()))
+
+    def test_append_activation_in_dygraph_global_use_mkldnn(self):
```

Comment:
LGTM for the tests to check the accuracy of the result when `use_mkldnn` is True and False.
I wonder if there are another ways that we checked the `mkldnn` is really used. For example, we can add some VLOG in the mkldnn kernel, and see if the log is printed when `fluid.set_flags({'FLAGS_use_mkldnn': True})` is called. And we can do it manually it is difficult for unittest.
Maybe you have already done that, I just remind.

Reply:
The problem we had is that AFAIK there is no better way to check if mkldnn kernel has been run than to check the output of DNNL_VERBOSE for now. The other idea is to run kernel throwing Exception with extended information about type of kernel. The other problem is that AFAIK python unittest lib does not have parsing c++ stdout result out of the box. Do you have suggestion what is the best way to check what type of kernel has been run from the python level ?

Reply:
> Do you have suggestion what is the best way to check what type of kernel has been run from the python level

I think checking the output of DNNL_VERBOSE is Ok. You can do it in the next PR.